src/main/scala/org/deepdive/helpers/Helpers.scala

```diff
@@ -171,4 +172,23 @@ object Helpers extends Logging {
         s""" --silent -N -e '${query.replaceAll("'", "'\\\\''")}' """
     }
   }
+
+
+  def slugify(input: String): String = {
```

Comment:
It's used for use client hostname as part of the path of distributed package copies.